app/models/experiment_template.py

```diff
@@ -38,13 +46,15 @@ def initialize_files(self, dockerfile, pip_requirements, script):
         base_path.joinpath("script.py").write_text(script)
 
         # REANA
-        image_name = f"{EXPERIMENT_TEMPLATE_DIR_PREFIX}{self.id}"
-        repository_name = f"{settings.DOCKER_REGISTRY_URL}/{image_name}"
+        tag_name = f"{EXPERIMENT_TEMPLATE_DIR_PREFIX}{self.id}"
+        whole_image_name = (
```

Comment:
The logic of creating `image_tag` and `image_name` are scattered around the codebase in multiple places. If we don't have enough time to reconcile this, please at least add a TODO so we have it marked.